### ISSUE TITLE
ListView in html5 population very very heavy?

### ISSUE THREAD

**Issue by Justinfront** [ianharrigan/haxeui#219]
I tried the ListView in html5, it works but does not seem to mask properly and the creation time with long lists times out the browser, if I limit creation to 10 items per frame it does not time out it just absorbs all processing power, once setup it works fine ( apart from masking failing ).  So confused as to why it is so heavy to add items to the list? Also I was not having any luck with IE11, but it seemed to work fine for other openfl parts of my project.

I set it up after sorting my data in an enterframe loop. 

stage.addEventListener(Event.ENTER_FRAME, populateSlowly);
	function populateSlowly(event: Event ): Void {
		for ( i in 0...10 ) {
			if( groundCount < groundData.length ) groundList.dataSource.add( groundData[groundCount] );
			if ( upperCount < upperData.length ) upperList.dataSource.add( upperData[upperCount] );
			groundCount++;
			upperCount++;
		}
		trace( 'populateSlowly' );
		if ( upperCount > upperData.length -1 && groundCount > groundData.length - 1 ) {
				Toolkit.openPopup( { x: 580, width: 250, height: 300 } , function( root: Root ) {
				root.addChild( upperList );
				root.addChild( groundList );
				stage.removeEventListener(Event.ENTER_FRAME, populateSlowly);
			});	
		}
		
	}

I don't really follow openfl well enough to know if I can use faster setup for webgl or something but suspect this issue is not completely related to render tech.
I can make my own ListView, probably easier that looking at haxeui source, but wanted to check if you had any thoughts on html5 components in particular ListView.
Also even on flash target the compile time when adding haxeui seems excessive, especially if I only need one component? 
But haxeui seems very useful well done on all the hardwork.

**Comment by ianharrigan** [comment]
Yeah, there are masking issues with html5 target... still need to track them down and work out a minimal example.

As for the list... there is an issue you have it create items virtually (ie, it will resuse the list items and not create a new one for each item. Although, to be fair, you are only creating 10... :/ Seems another thing to check out... 

(Ill checkout the compile time also, might spread these out to seperate issues)

**Comment by ianharrigan** [comment]
(Sorry for the late reply - been trying to get another project closed off)

**Comment by Justinfront** [comment]
I looked at some js ones and there were some that worked really well, like scrolling spread sheets but they probably would not be so easy to integrate and they all seem to use jQuery :(.  I don't know if a luxe webgl implimentation would be more viable for js targeting, or maybe it will be faster when openfl has webgl target, have you looked at luxe at all you can draw directly to the gl.  Flambe does not seem to have proper access to drawing so I am guessing it's less suitable for a haxeui solution.
